nextcord/ext/commands/bot.py

```diff
@@ -901,6 +903,135 @@ def reload_extension(self, name: str, *, package: Optional[str] = None) -> None:
             # revert sys.modules back to normal and raise back to caller
             sys.modules.update(modules)
             raise
+    
+    def load_extensions(
+            self, 
+            names: List[str], 
+            *, 
+            package: Optional[str] = None,
```

Comment:
Possible I missed this, but where is this used in the method?

Reply:
Where is what used in the method? The `package` attribute?

Reply:
As discussed on Discord, this was actually an oversight on my part.